src/textures/DepthTexture.js

```diff
@@ -28,9 +28,31 @@ class DepthTexture extends Texture {
 		this.flipY = false;
 		this.generateMipmaps = false;
 
+		this.compareFunction = null;
+
 	}
 
 
+	copy( source ) {
+
+		this.compareFunction = source.compareFunction;
+
+		super.copy( source );
```

Comment:
Can we switch the order of both statements? So that the `super` call comes first.

Reply:
Done!